source/interaction/interaction_utilities.cc

```diff
@@ -202,7 +204,6 @@ namespace fdl
   }
 
 
-
```

Comment:
Can you get rid of these extra whitespace changes?

Reply:
this is better?